shell/app/modules/application/stores/repo.ts

```diff
@@ -583,6 +585,14 @@ const repoStore = createStore({
       });
       update({ mrStats });
     },
+    async getAppMRStatsCount({ call, update }) {
```

Comment:
is it required to be a store function? what's the reason not use api call directly